cartographer/mapping/proto/sparse_pose_graph_options.proto

```diff
@@ -47,4 +47,7 @@ message SparsePoseGraphOptions {
   // Rate at which we sample a single trajectory's scans for global
   // localization.
   optional double global_sampling_ratio = 5;
+
+  // Whether to calculate and output histograms for the pose residuals.
```

Comment:
Drop the `calculate and`? Sounds like an implementation detail. Please run `scripts/update_configuration_doc.py`.